h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostJavaMojoModel.java

```diff
@@ -29,14 +27,23 @@
   private TreeSHAPPredictor<FVec> _treeSHAPPredictor;
   private OneHotEncoderFactory _1hotFactory;
 
+  @Deprecated
   public XGBoostJavaMojoModel(byte[] boosterBytes, String[] columns, String[][] domains, String responseColumn) {
-    this(boosterBytes, columns, domains, responseColumn, false);
+    this(boosterBytes, null, columns, domains, responseColumn, false);
```

Comment:
What about empty array instead of null?

Reply:
That would need special handling in a bunch of places (model creation, old MOJO import, interpreting of bytes). I don't see an immediate benefit of doing that.

Reply:
I agree